tests/test_wrappers.py

```diff
@@ -0,0 +1,251 @@
+import pytest
+import pandas as pd
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.preprocessing import StandardScaler
+from feature_engine.wrappers import SklearnTransformerWrapper
+import numpy as np
+
```

Comment:
I've read that imports are normally done alphabetically. can we re-organise? also from sklearn.preprocessing, would it be nicer if we have the 2 imports in 1 line?

Reply:
good point, it's PEP8 stuff. I'll update this in next commit (and as it's in PEP8, there will be two groups - third party libraries and local library import)